cli/prisma2/src/bin.ts

```diff
@@ -120,21 +120,38 @@ if (require.main === module) {
         process.exit(code)
       }
     })
-    .catch(err => {
-      if (err.rustStack) {
-        handlePanic(err, packageJson.name, packageJson.version).catch(e => {
+    .catch((err) => {
+      let shouldExit = false
+
+      function handleIndividualError(err) {
```

Comment:
This introduces **shadowing** of the variable `err`. Please name it differently in the `handleIndividualError` method.

Reply:
Indeed! Will fix asap :)